artman/tasks/protoc_tasks.py

```diff
@@ -240,7 +240,7 @@ class RubyGrpcCopyTask(task_base.TaskBase):
     the gapic_code_dir/lib.
     """
     def execute(self, api_name, api_version, language, organization_name,
-                output_dir, gapic_code_dir, grpc_code_dir):
+                output_dir, pkg_dir, gapic_code_dir, grpc_code_dir):
```

Comment:
I added this to try and solve the `NameError` farther down, but the problem is deeper. It does need to be there though.